package.json

```diff
@@ -25,7 +25,13 @@
     "body-parser": "^1.19.0",
     "clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service",
     "dotenv-flow": "^3.2.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "fhirclient": "^2.3.1",
+    "fhirpath": "^2.3.0",
+    "fs": "0.0.1-security",
```

Comment:
The `fs` package does not really exist, it exists in NPM solely to prevent people from accidentally installing a malicious package that replaces that built-in `fs` module.